sjb/actions/repo_sync.py

```diff
@@ -12,9 +12,8 @@
         </hudson.model.StringParameterDefinition>""")
 
 _SYNC_TITLE_TEMPLATE = Template("SYNC {{ repository | upper }} REPOSITORY")
-_SYNC_ACTION_TEMPLATE = Template("""oct sync remote {{ repository }} --branch "${{ '{' }}{{ repository | replace('-', '_') | upper }}_TARGET_BRANCH}" """)
-
-_SYNC_DESCRIPTION_TEMPLATE = Template("""Using the <a href="https://github.com/openshift/{{ repository }}/tree/${{ '{' }}{{ repository | replace('-', '_') | upper }}_TARGET_BRANCH}">{{ repository }} ${{ '{' }}{{ repository | replace('-', '_') | upper }}_TARGET_BRANCH}</a> branch.""")
+_SYNC_ACTION_TEMPLATE = Template("""oct sync remote {{ repository }} --branch "${{ '{' }}{{ repository_target_branch | replace('-', '_') | upper }}_TARGET_BRANCH}" """)
```

Comment:
yeah thats what the var is for.. sorry my brain was hurting when I was writing it :face_with_head_bandage:  ... will update the name and class docs